package-system/AWSNativeSDK/README.md

```diff
@@ -0,0 +1,101 @@
+# Build New AWS Service Target in AWSNativeSDK
+First check package directories at [aws/aws-sdk-cpp](https://github.com/aws/aws-sdk-cpp) on GitHub. 
+The package name is the suffix of the directory name for the service.
+
+```
+aws-sdk-cpp\aws-cpp-sdk-<packageName>   # Repo directory name and packageName
+aws-sdk-cpp\aws-cpp-sdk-s3              # Example: Package name is s3
+```
+
+Note: If your target service is not available in current AWSNativeSDK version (but it is listed in aws-sdk-cpp repo),
+then please upgrade AWSNativeSDK version first.
+
+## Example: Build ElasticSearch (ES)
+
+### 1. Check [aws/aws-sdk-cpp](https://github.com/aws/aws-sdk-cpp) for aws-cpp-sdk-es
+
+### 2. Add ElasticSearch as part of AWSNativeSDK build target
+Modify AWSNativeSDK build script by adding ElasticSearch as a build target, use windows build script as a reference:
```

Comment:
Nit: "Modify the AWSNativeSDK build script by adding ElasticSearch as a build target. The examples below use the windows build script as a reference:"

Reply:
done